### DISCUSSION TITLE
Adding a Serial. println() method facilitates debugging

### DISCUSSION THREAD

**Discussion by peterwillcn** [micropython/micropython#11123] [RP2040 / Pico]
issue: https://github.com/scientifichackers/ampy/issues/118
![debug](https://user-images.githubusercontent.com/323794/227719993-917f3db5-966d-497c-b7f9-e187ff056020.jpg)

**Comment by peterwillcn** [comment]
print("Temperature: ", temperature) ====>    Serial. println("Temperature: ", temperature) ?

**Comment by scruss** [comment]
If you've already got /dev/ttyACM0 open in minicom, ampy won't be able to interact with the board since the serial port is in use.

**Reply by peterwillcn** [reply]
Thank you very much @scruss
Correct. Close minicom, and the print() method can work properly. This is due to my incorrect operation sequence. 
![Screenshot 2023-03-25 at 22 18 39](https://user-images.githubusercontent.com/323794/227722812-93988447-0103-4f2e-b604-7635cbe958b5.png)
